.config.yml

```diff
@@ -18,7 +18,7 @@ app_host: 'http://localhost:3000'
 driver: chrome
 
 # Runs either Chrome or Firefox driver headlessly if set to true
-headless: true
+headless: false
```

Comment:
Are you sure you want to commit this change?

Reply:
Thanks Cinti! I'll leave this as is for now, as I regularly switch between headless and non-headless.